package.json

```diff
@@ -116,10 +106,12 @@
     "pretest": "npm run hint"
   },
   "devDependencies": {
-    "babel-core": "6.26.0",
+    "babel-cli": "^6.26.0",
```

Comment:
can you remove the carets? ^

Reply:
@xdamman yes, but I would like to have a discussion about that! Just to make clear why we don't want them and write that somewhere for posterity.

My personal taste is to use them by default, because it's easier to get the updates (bug fix and security) and we can rely on `package-lock.json` to have reproducible deployments.

In the end, the important thing is to be consistent, with or without carets!

Reply:
Agreed on consistency.
The reason was to avoid bad surprises when deploying like we had in the past (but I think it was before the introduction of `package-lock`). It's very hard to debug when you don't know which dependency is creating an issue.
@asood123 had a strong opinion about it. I don't.